memgpt/embeddings.py

```diff
@@ -177,10 +177,6 @@ def embedding_model(config: EmbeddingConfig, user_id: Optional[uuid.UUID] = None
             api_version=config.azure_version,
         )
     elif endpoint_type == "hugging-face":
-        try:
-            embed_model = EmbeddingEndpoint(model=config.embedding_model, base_url=config.embedding_endpoint, user=user_id)
```

Comment:
Note: removed because it's probably not a good idea to silently error on embedding endpoint failures